Common Functions/splice-functions/pom.xml

```diff
@@ -0,0 +1,25 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>splice</groupId>
+  <artifactId>splice-functions</artifactId>
```

Comment:
I'm wondering if we want to call this splice-functions (for the artifact and the name).  This is something custom and I don't think we should confuse it with 'base code'.